src/hooks/balances/lib/getBalancesByAddress.test.tsx

```diff
@@ -0,0 +1,148 @@
+import { ApiPromise } from '@polkadot/api';
+import { AccountId32 } from '@polkadot/types/interfaces';
+import { Balance } from '../../../generated/graphql';
+import constants from '../../../constants';
+import {
+  getBalancesByAddress,
+  fetchNativeAssetBalance,
+} from './getBalancesByAddress';
+
+export const nativeAssetBalance = '10';
+export const nonNativeAssetBalance = '20';
+
+export const getMockApiPromise = () =>
+  ({
+    query: {
+      system: {
+        account: jest.fn((arg: AccountId32 | string | Uint8Array) => {
+          return {
+            data: {
+              free: nativeAssetBalance,
+            },
+          };
+        }),
+      },
+      tokens: {
+        accounts: {
+          multi: jest.fn((arg: (unknown[] | unknown)[]) => {
+            return arg.map((arg) => {
+              return { free: nonNativeAssetBalance };
+            });
+          }),
+        },
+      },
+    },
+  } as unknown as ApiPromise);
+
+describe('hooks/balances/lib/getBalancesByAddress', () => {
```

Comment:
i'd rather keep it simple e.g. `getBalanceByAddress`, the full file path will be provided by the test runner anyways